### ISSUE TITLE
Want to display XQuery analysis report on SonarQube and not on command line

### ISSUE THREAD

**Issue by mohiteshweta** [malteseduck/sonar-xquery-plugin#25]
I am using the XQuery plugin 2.0 created according to the steps mentioned by malteseduck. I added the plugin to "SonarqQube/extentions/plugins" folder. Tested the Xquery code using sonar scanner. I can see the error, issues as INFO on the command line. My project demands to display the errors/issues on SonarQube dashboard. Is there any setting to be done to direct the analysis report to SonarQube server ? or is there any other way I can display the analysis report on SonarQube dashboard.

**Comment by nanthakumarv** [comment]
Yes, is there a solution for this. I am too facing the same issue. The errors are being reported only in the console, but in the dashboard we see zero results.
The xqtest-reports is not getting generated and hence the dashboard displays zero results.
[sonar:sonar] 12:12:45.140 INFO  - Sensor org.sonar.plugins.xquery.XQuerySensor@7bbcf6f0 done: 2608 ms
[sonar:sonar] 12:12:45.141 INFO  - Sensor XQueryTestSensor...
[sonar:sonar] 12:12:45.142 INFO  - parsing target\xqtest-reports
[sonar:sonar] 12:12:45.142 WARN  - Reports path not found: C:\Users\vv\Documents\testXquery\Build_Src\target\xqtest-reports

Can you please let know a solution for this.?
